Source/m_menu.c

```diff
@@ -3021,7 +3014,6 @@ static const char *weapon_attack_alignment_strings[] = {
 setup_menu_t weap_settings1[] =  // Weapons Settings screen       
 {
   {"ENABLE RECOIL", S_YESNO,m_null,WP_X, WP_Y+ weap_recoil*8, {"weapon_recoil"}},
-  {"ENABLE BOBBING",S_YESNO,m_null,WP_X, WP_Y+weap_bobbing*8, {"player_bobbing"}},
```

Comment:
So, this leaves Boom recoil as the only non-cosmetic option in this menu? Hm... do you still have the branch that removes it? 🙇

Reply:
There is also Classic BFG:
![doom00](https://user-images.githubusercontent.com/5077629/165238368-07ff5d43-a345-4fef-9faf-8f4dc1c47667.png)

Reply:
> There is also Classic BFG

Yes, right, that's a special pony. The only hard game-breaking option that we face to the user, right?

Reply:
If you ask me, I'd like to remove Classic BFG and Boom recoil from the menu... and many other options.

Reply:
I agree the menu is still not perfect, but we have improved a lot already.

I'd like to keep the Classic BFG for now, but agree that "Enable Recoil" has to go.

"Why should I care what I said yesterday" - Konrad Adenauer (roughly) 🤷 😁